src/api/index.js

```diff
@@ -96,13 +99,15 @@ export async function getMentors() {
   return res.data;
 }
 
-export async function createApplicationIfNotExists() {
-  const myApplications = await makeApiCall(`${paths.MENTORS}/myapplications?status=pending`);
+export async function createApplicationIfNotExists(user) {
+  const myApplications = await makeApiCall(
```

Comment:
Is the url should contains `?status=pending`?

Reply:
you are right, I thought that the new endpoint will replace the previos one completely

Reply:
I'm not sure about that. Can you check please? I'm afk now.

Reply:
I reviewed the backend code, it is mandatory

Reply:
@crysfel could you confirm this please